src/main/java/de/tum/in/www1/artemis/repository/ParticipationRepository.java

```diff
@@ -49,8 +49,9 @@ default Participation findByIdWithLatestSubmissionElseThrow(Long participationId
     }
 
     @Query("""
-            SELECT p FROM Participation p
-            LEFT JOIN FETCH p.submissions s
+            SELECT p
+            FROM Participation p
+                LEFT JOIN FETCH p.submissions s
             WHERE p.id = :#{#participationId}
```

Comment:
Let's directly change this while touching this statement.
```suggestion
            WHERE p.id = :participationId
```